packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -22,20 +22,32 @@ export const pageViewTrackingPlugin: CreatePageViewTrackingPlugin = (options: Op
   let localConfig: BrowserConfig;
   const { trackOn, trackHistoryChanges, eventType = defaultPageViewEvent } = options;
 
+  const getDecodeURI = (locationStr: string): string => {
+    let decodedLocationStr = locationStr;
+    try {
+      decodedLocationStr = decodeURI(locationStr);
```

Comment:
Use decodeURI instead of decodeURIComponent since we are not decoding special characters like ?, #, etc. In the browser's navigation bar, these special characters are also not decoded.

For example, if the link provided is www.example.com/home-shell%3Fx%3Dtest, %3Fx%3Dtest represents ?x=test. The window.location.href will still be www.example.com/home-shell%3Fx%3Dtest, which matches how it appears in the navigation bar.